rocket_sqlite_oracle/src/db/kraken.rs

```diff
@@ -0,0 +1,380 @@
+use anyhow::{anyhow, Context, Result};
+use futures::{SinkExt, StreamExt, TryStreamExt};
+use rocket::tokio::sync::watch;
+use rust_decimal::prelude::*;
+use serde::Deserialize;
+use std::convert::{Infallible, TryFrom};
+use std::sync::Arc;
+use std::time::{Duration, SystemTime};
+use url::Url;
+
+/// Connect to Kraken websocket API for a constant stream of rate updates.
```

Comment:
Why is this file in the `db` module? Has little to do with the database.

Reply:
This is here because the `db` module contains the `kraken_ticker` function. Hypothetically, if one were to add more tickers then they could be added to `src/db/` in a consistent manner.

Reply:
But what does the kraken ticker have semantically to do with the database? This file should be in `rocket_sqlite_oracle/src/kraken.rs`.